cli.js

```diff
@@ -777,6 +784,39 @@ async function powersOfTauImport(params, options) {
     // TODO Verify
 }
 
+async function powersOfTauImportNoOrigin(params, options) {
+    let curveName;
+    let power;
+    let response;
+    let newPtauName;
+    let importPoints = true;
+    let doCheck = true;
+
+    curveName = params[0];
+
+    power = parseInt(params[1]);
+    if ((power<1) || (power>28) || isNaN(power)) {
+        throw new Error("Power must be between 1 and 28");
+    }
+
+    response = params[2];
+    newPtauName = params[3];
+
+    if (options.nopoints) importPoints = false;
+    if (options.nocheck) doCheck = false;
+
+    if (options.verbose) Logger.setLogLevel("DEBUG");
+
+    const curve = await curves.getCurveFromName(curveName);
+
+    const res = await powersOfTau.importResponseNoOrigin(curve, power, response, newPtauName, options.name, importPoints, logger);
+
+    if (res) return res;
+    if (!doCheck) return;
+
+    // TODO Verify
```

Comment:
What should be verified here?

Reply:
This results from a direct copy of function powersOfTauImport. No need to verify anything here. Removed this comment in the [new commit](https://github.com/MystenLabs/snarkjs/pull/1/commits/b5f6cf3ba828f3fbb8cf4100e7a1c1ec6408d7d9#diff-180b3d2af89416b99563fff8c37eb8ca1ef7591fc1ddd736b5dacf740ccd3760L816-L817).

Reply:
Okay. Do you know what the TODO in the original code refers to?

Reply:
Maybe to check the validity of the imported contribution. They only left this comment with no other info. We cannot do the ordinary verification for the imported contributions anyway. Instead, we can do the simple sanity check by exporting the resulting ptau file to kobi-compatible challenge file and compare it with the original challenge file bit by bit.